site/src/xServices/users/usersXService.ts

```diff
@@ -57,6 +57,8 @@ export interface UsersContext {
   updateUserRolesError?: Error | unknown
   paginationContext: PaginationContext
   paginationRef: PaginationMachineRef
+  count: number
+  getCountError: Error | unknown
```

Comment:
thank you, I meant to make sure all the errors were handled! though it might be okay for this to fail silently...maybe a console error is the best option, what do you think?

Reply:
Yeah, I completely agree! No need to warn the user but a console.error for us devs might be nice.